.gitignore

```diff
@@ -1,5 +1,11 @@
 *~
-topologies/kne/testbed.kne.yml
 .vscode/
 # used by `make validate_paths`
-openconfig_public/
\ No newline at end of file
+openconfig_public/
+launch.json
```

Comment:
Aren't launch.json and settings.json already covered by .vscode/?

Reply:
...removed both....json files are part of .vscode/